nemo/core/neural_modules.py

```diff
@@ -8,9 +8,11 @@
 from collections import namedtuple
 from enum import Enum
 from inspect import getargvalues, getfullargspec, stack
+from os import path
 from typing import Dict, List, Optional, Set, Tuple
 
-import nemo
+import yaml
```

Comment:
Why yaml? ASR defaults to from ruamel.yaml import YAML

Reply:
good catch, thx!